VA-Explorer/va_explorer
# Export api

This request primarily closes #167 by adding an api to download data, a download form view that connects to it, and a suite of tests to validate download functionality

To test: 
Go to Download Data tab, fill out form in a way that filters data, download, and confirm data meets filter criteria

Also closes #157
